language/move-prover/doc/user/install.md

```diff
@@ -13,6 +13,10 @@ run (in the Libra root directory):
 
 This command should work on MacOS and Linux flavors like Ubuntu or CentOS. (Windows is currently not supported).
 
+Notice that you have to include environment variable definitions in `~/.profile` into your shell. Depending on your
+setup, the  `~/.profile` may be already automatically loaded for each login shell, or it may not. If not, you may
+need to add `. ~/.profile` to your `.bashrc` or other shell configuration manually.
```

Comment:
On my machine .bashrc is ignored; instead .bash_profile is looked at.  You might want to mention .bash_profile in this comment as well.

Reply:
Done